mods/digimon/scripts.js

```diff
@@ -0,0 +1,125 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ *  Digimon Showdown                                     *
+ *  Created By:                                          *
+ *  Insist + Ashley the Pikachu + Stellation + AlfaStorm *
+ * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+'use strict';
+
+exports.BattleScripts = {
+    randomDigimonTeam: function (side) {
+        let team = [];
+        var variant = (this.random(2) === 1);
+        var sets = {
+            "Botamon": {
+                species: "Botamon",
+                ability: "Data",
+                moves: ['bubble'],
+                signatureMove: "Protect",
+                nature: "Serious",
+            },
```

Comment:
Eh, tbh I feel way more comfortable with coding it in scripts.js (probably because I code these there a lot more)